plugins/postcss-font-format-keywords/README.md

```diff
@@ -0,0 +1,106 @@
+# PostCSS Font Format [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][postcss]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-font-format.svg" height="20">][npm-url]
+[<img alt="CSS Standard Status" src="https://cssdb.org/images/badges/font-format-keywords.svg" height="20">][css-url]
+[<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url]
+[<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Font Format Keywords] lets you specify font formats as keywords, following the [CSS Fonts] specification.
+
+```pcss
+@font-face {
+  src: url(file.woff2) format(woff2);
+}
+
+/* becomes */
+
+@font-face {
+  src: url(file.woff2) format("woff2");
+}
+```
+
+_See prior work by [valtlai](https://github.com/valtlai) here [postcss-font-format-keywords](https://github.com/valtlai/postcss-font-format-keywords)
+To ensure long term maintenance and to provide the needed features this plugin was recreated based on valtlai's work._
+
+## Usage
+
+Add [PostCSS Font Format Keywords] to your project:
+
+```bash
+npm install postcss @csstools/postcss-hwb-function --save-dev
+```
+
+Use it as a [PostCSS] plugin:
+
+```js
+const postcss = require('postcss');
+const postcssFontFormatKeywords = require('@csstools/postcss-hwb-function');
+
+postcss([
+  postcssFontFormatKeywords(/* pluginOptions */)
+]).process(YOUR_CSS /*, processOptions */);
+```
+
+[PostCSS Font Format Keywords] runs in all Node environments, with special
+instructions for:
+
+| [Node](INSTALL.md#node) | [PostCSS CLI](INSTALL.md#postcss-cli) | [Webpack](INSTALL.md#webpack) | [Create React App](INSTALL.md#create-react-app) | [Gulp](INSTALL.md#gulp) | [Grunt](INSTALL.md#grunt) |
+| --- | --- | --- | --- | --- | --- |
+
+## Options
+
+### singleQuotes
+
+The `singleQuotes` option determines whether to use single quotes. By default double quotes `"` are used.
+
+```js
+postcssFontFormatKeywords({ singleQuotes: true })
+```
+
+```pcss
+@font-face {
+  src: url(file.woff2) format(woff2);
+}
+
+/* becomes */
+
+@font-face {
+  src: url(file.woff2) format('woff2');
+}
+```
```

Comment:
Is there a technical reason to have this option?
If there isn't, I would remove it.

Reply:
Other than, it was already there, there's no other reason